.github/ISSUE_TEMPLATE/QUESTION.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I know we want to discourage people using GitHub issues as QA platform.

Should we turn this into FAQs too? Like, IE11 ES5, etc.

Reply:
https://github.com/Microsoft/BotFramework-WebChat/issues/1961